recipes/inotify-tools/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "inotify-tools" %}
+{% set version = "3.14" %}
+{% set sha256 = "97a4fdf9132da47c2baa677c1a427c435ee55a26df42b2b74a41489b9fbd2323" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/rvoicilas/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+
+test:
+  commands:
+    - inotifywatch --help > /dev/null || [[ $? == 1 ]]
```

Comment:
What is the `[[ $? == 1 ]]` accomplishing here? Doesn't `conda` fail if `$?` is any nonzero value, i.e. the same situations as the `||` branch would be taken?

Reply:
Incidentally, this is actually what's causing the linter to complain, which is a bug in the linter (https://github.com/conda-forge/conda-smithy/issues/576). But I think you should probably drop it anyway. :)

Reply:
Actually, `inotifywatch --help` returns `1` so I have no other way to test the `inotifywatch` execution (conda fails if one of the test commands returns a non zero status). I could test `which inotifywatch` but the program itself is not executed.